applications/accounts/web/src/styles.css

```
@@ -0,0 +1,4 @@
+@import '../../../../packages/breeze-ui/src/styles/reset.css';
+@import '../../../../packages/breeze-ui/src/styles/styles.css';
+
+@source './**/*.{ts,tsx}';
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@source`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)